datasync/fetcher/doc.go

```diff
@@ -18,5 +18,15 @@
 // This file is derived from eth/fetcher/fetcher.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package fetcher contains the block announcement based synchronisation.
+/*
+Package fetcher contains the block announcement based synchronisation.
+ How fetcher works
+ Insert the received block into the chain and broadcast a block or block hash to the peer.
+If a block hash is received instead of a block, node requests and reflects the header and body from the connected peer.
```

Comment:
How about change the line like below?

```suggestion
If a block hash is received instead of a block, node requests and reflects the header and body from the peer who sent the block hash.
```